gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -364,6 +364,24 @@ message TripUpdate {
     extensions 9000 to 9999;
   }
   optional TripProperties trip_properties = 6;
+  
+  // Realtime occupancy immediately after departure for a given stop on a
+  // trip. Occupancies can be supplied for both past and future events.
+  // The producer is allowed, although not required, to drop past events.
+  message StopOccupancy {
+    // The update is linked to a specific stop either through stop_sequence or
+    // stop_id, so one of the fields below must necessarily be set.
+
+    // Must be the same as in stop_times.txt in the corresponding GTFS feed.
+    optional uint32 stop_sequence = 1;
+    // Must be the same as in stops.txt in the corresponding GTFS feed.
+    optional string stop_id = 2;
+
+    // Expected occupancy after departure from the given stop.
+    optional VehiclePosition.OccupancyStatus occupancy_status = 3;
```

Comment:
tl;dr - There is a lot of discussion if OccupancyStatus is the right way forward for occupancy. See https://github.com/google/transit/issues/223 for discussion vs occupancy_percentage and an older proposal to revise OccupancyStatus at https://github.com/google/transit/pull/212. There is also a newer draft proposal at https://github.com/TransitApp/transit/pull/3 to define a fixed linear scale for occupancy, which could deprecate OccupancyStatus.

All or some of these (OccupancyStatus, occupancy_percentage, CrowdLevel) could be used here.

Reply:
That's fine, the intention here is to use the standard unit of occupancy that applies to the rest of GTFS, whatever that may change to in the future.